vertx-mssql-client/src/main/java/io/vertx/mssqlclient/MSSQLConnection.java

```diff
@@ -84,56 +81,25 @@ static void connect(Vertx vertx, MSSQLConnectOptions connectOptions, Handler<Asy
   @Override
   MSSQLConnection preparedQuery(String s, Handler<AsyncResult<RowSet<Row>>> handler);
 
-  /**
-   * {@inheritDoc}
-   */
-  @Fluent
-  @GenIgnore
-  @Override
-  <R> MSSQLConnection preparedQuery(String s, Collector<Row, ?, R> collector, Handler<AsyncResult<SqlResult<R>>> handler);
```

Comment:
can these changes really be ported to 3.9? Isn't this a breaking change that would need to wait until 4.0?

Reply:
yes it is breaking but in 3.x SQL client is in tech preview mode and can break.